layer/Icon.Canvas.js

```diff
@@ -1,4 +1,4 @@
-L.Icon.Canvas = L.Icon.extend({
+module.exports = L.Icon.Canvas = L.Icon.extend({
```

Comment:
Won't will break in a browser that does not support es6 modules (currently most of them), since module is not defined ?